src/zulip.rs

```diff
@@ -188,6 +189,12 @@ fn handle_command<'a>(
                                 .await
                                 .map_err(|e| format_err!("Failed to await at this time: {e:?}"))
                             }
+                            Some("ping-goals") => {
```

Comment:
So...I feel like it may be prudent to gate this so that only you (@nikomatsakis) can run this.

Reply:
...good idea, at least for now. :)